src/app/workflow/view/view.component.html

```diff
@@ -56,6 +56,19 @@
     >
       Request DOI
     </button>
+    <button
+      mat-menu-item
+      id="OrcidButton"
+      color="accent"
+      class="btn-block"
```

Comment:
that bootstrap 3?

Reply:
Looks like; I copied it from an it's the same as all the other items on that menu. Removed from all, looks the same as with.